riotfile.py

```diff
@@ -1665,6 +1665,28 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="pytest-benchmark",
+            command="pytest {cmdargs} tests/contrib/pytest_benchmark/",
+            pkgs={"msgpack": latest},
+            venvs=[
+                Venv(
+                    pkgs={
+                        "more_itertools": "<8.11.0",
```

Comment:
Is this a `pytest-benchmark` dependency? I don't see the plugin or tests using it.